manifests/cirunner/docker.pp

```diff
@@ -0,0 +1,70 @@
+# == Class: gitlab::cirunner::docker
+#
+class gitlab::cirunner::docker (
+  Boolean                    $manage_docker                     = true,
+  String                     $xz_package_name                   = 'xz-utils',
+  String                     $default_image                     = 'ubuntu_trusty',
+  Hash                       $docker_images                     = {},
+  Hash                       $runners                           = {},
+  Optional[String]           $default_token                     = undef,
+  Optional[String]           $default_url                       = undef,
+  Optional[Boolean]          $default_run_untagged              = undef,
```

Comment:
is it possible that the booleans are not Optional but default to true/false? That often makes sense (not always) and is then the preferred solution.

Reply:
would need to double check but i think . making them all false would help be fine

Reply:
these are correct as is. Please don't change them.